x-pack/plugin/ccr/qa/rest/src/test/resources/rest-api-spec/test/ccr/auto_follow.yml

```diff
@@ -1,30 +1,49 @@
 ---
 "Test put and delete auto follow pattern":
+  - do:
+      cluster.state: {}
+
+  - set: {master_node: master}
+
+  - do:
+      nodes.info: {}
+
+  - set: {nodes.$master.transport_address: local_ip}
+
+  - do:
+      cluster.put_settings:
+        body:
+          transient:
+            cluster.remote.local.seeds: $local_ip
+        flat_settings: true
+
+  - match: {transient: {cluster.remote.local.seeds: $local_ip}}
+
   - do:
       ccr.put_auto_follow_pattern:
-        leader_cluster_alias: _local_
+        leader_cluster_alias: local
```

Comment:
Can we change this to `leader_cluster` in a follow-up?

Reply:
I made the change, because this was a trivial change: 56d4f6971845fcf30f08be70f81ade1e4fe3d6b8